internal/backoff/context_test.go

```diff
@@ -1,14 +0,0 @@
-// Copyright (C) 2019-2023 vdaas.org vald team <vald@vdaas.org>
```

Comment:
this file needs to be exists to include context.go in code coverage report

Reply:
There is no test in this file so I deleted it.
And if test template update we can re-generate it.